include/transmem/transmem.h

```diff
@@ -40,9 +41,9 @@ class NoSuchLinkFoundException : public std::runtime_error {
     FrameID destFrame;
 };
 
-/***************************
- * EMPTYLINKQUERYEXCEPTION *
- ***************************/
+/******************************
+ * EMPTY LINK QUERY EXCEPTION *
```

Comment:
I will comment all the code of my product with doxygen comments in the end. I though it was a good idea to wait, since the code changes frequently during the development and i don't wanted to waste to much time with that.